py3status/parse_config.py

```diff
@@ -632,19 +685,23 @@ def remove_any_contained_modules(module):
                 fixed[k] = v
         return fixed
 
+    def append_modules(item):
```

Comment:
This was just a little bit of code cleanup I noticed :)

Reply:
okay for this time, but please separate these kind of PRs next time

Reply:
I removed it and moved it to a new PR! :)